### PR TITLE
support JDK12+ signature tests.

### PR DESCRIPTION
**Pull request by gurunrao**
support JDK12+ signature tests.
Signed-off-by: gurunandan.rao@oracle.com <gurunandan.rao@oracle.com>

### CODE REVIEW

File: docker/build_jaxbtck.sh [hunk 1]
```diff
@@ -19,19 +19,19 @@ if [ -z "$ANT_HOME" ]; then
...
 if [ -z "$JAVA_HOME" ]; then
-  export JAVA_HOME=/opt/jdk1.8.0_171
+  export JAVA_HOME=/opt/openjdk-8u292-b10
 fi
 
 export PATH=$JAVA_HOME/bin:$ANT_HOME/bin:$PATH
 
 if [ -z "${JAXB_RI_BUNDLE_URL}" ]; then
-  export JAXB_RI_BUNDLE_URL='https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/3.0.0-M4/jaxb-ri-3.0.0-M4.zip'
+  export JAXB_RI_BUNDLE_URL='https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/3.0.0/jaxb-ri-3.0.0.zip'
```
Comment (lukasj) [inline docker/build_jaxbtck.sh:28]: why not 3.0.1?

File: docker/build_jaxbtck.sh [hunk 2]
```diff
@@ -19,19 +19,19 @@ if [ -z "$ANT_HOME" ]; then
...
 fi
 if [ -z "${JAF_BUNDLE_URL}" ];then
-  export JAF_BUNDLE_URL='https://repo1.maven.org/maven2/jakarta/activation/jakarta.activation-api/2.0.0-RC3/jakarta.activation-api-2.0.0-RC3.jar'
+  export JAF_BUNDLE_URL='https://repo1.maven.org/maven2/jakarta/activation/jakarta.activation-api/2.0.0/jakarta.activation-api-2.0.0.jar'
```
Comment (lukasj) [inline docker/build_jaxbtck.sh:31]: 2.0.1?

**Review by lukasj — APPROVED** [review]
LGTM

**Review by lukasj — COMMENTED** [review]
btw: would it be possible to also add JDK-17-latest and JDK-latest options (see https://wiki.eclipse.org/Jenkins#JDK)  to run TCK on to avoid a need of the new PR per JDK in the future?

**Comment by gurunrao** [comment]
> btw: would it be possible to also add JDK-17-latest and JDK-latest options (see https://wiki.eclipse.org/Jenkins#JDK) to run TCK on to avoid a need of the new PR per JDK in the future?

Currently JDK is picked up from docker images, we are planning to have JDK configured at Jenkins jobs for future release. Scripts handling JDK are not part of TCK, hence they can be updated post release.

**Comment by lukasj** [comment]
I see, thx

**Pull request merged**